css/styles.css

```diff
@@ -40,3 +42,16 @@ textarea:focus {
 .value-changed {
   background-color: rgb(231, 240, 254);
 }
+
+.nav-bar {
+  align-items: center;
+  gap: 10px;
+}
+
+.pseudo-button {
+  height: 2rem;
+  text-align: center;
+  line-height: calc(2rem - 4px);
+  padding: 1px 6px;
+  font-size: calc(40px / 3);
```

Comment:
This is fine here, but overall there's not really a need to try and make these very specific sizes for font and line height. I would recommend just picking something that is close and using it whether it be a line height of 1.5-1.75rem and a font size of 13px